hs_core/hydroshare/resource.py

```diff
@@ -77,7 +77,11 @@ def get_science_metadata(pk):
     Exceptions.NotFound  - The resource identified by pid does not exist
     Exception.ServiceFailure  - The service is unable to process the request
     """
-    return get_system_metadata(pk)
+    res = utils.get_resource_by_shortkey(pk)
+    if res:
+        return res.metadata.get_xml()
+
+    #return get_system_metadata(pk)
```

Comment:
this returns None in some cases? If no resource is returned, shouldn't this be a NotFound?